modules/virt-template-vmi-probe-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-template-vmi-probe-config_{context}"]
+= Template: Virtual machine instance configuration file for defining liveness and readiness probes
+
+[source,yaml]
+----
+  apiVersion: kubevirt.io/v1alpha3
+  kind: VirtualMachineInstance
+  metadata:
+    labels:
+      special: vmi-fedora
+    name: vmi-fedora
+  spec:
+    domain:
+      devices:
+        disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+        - disk:
+            bus: virtio
+          name: cloudinitdisk
+      resources:
+        requests:
+          memory: 1024M
+    livenessProbe:
+      initialDelaySeconds: 120
+      periodSeconds: 20
+      tcpSocket: 
+        host: <host_IP>
```

Comment:
I think in this case the host is not needed. It should always default to the Pod IP. So only port should be specified.
In this case probably `port: 1500` as well to make this work as it will then try to TCP the same endpoint that provides the readinessProbe.
In general, I'd honestly not include the livenessProbe. Still think it's unlikely for people to use it a lot with VMIs and if the user tries on a slow system their VM might just get stuck in a crash loop.

Reply:
Okay, I have removed that portion from the template.